plugins/module_utils/network/nxos/facts/legacy/base.py

```diff
@@ -162,6 +162,47 @@ def populate(self):
                 self.facts["memtotal_mb"] = self.parse_memtotal_mb(data)
                 self.facts["memfree_mb"] = self.parse_memfree_mb(data)
 
+        data = None
+        data = self.run("show processes cpu | include CPU")
+
+        if data:
+            self.facts["cpu_utilization"] = self.parse_cpu_utilization(data)
+
+    def parse_cpu_utilization(self, data):
+        facts = dict()
```

Comment:
Just one of my pet peeves. :)
```suggestion
        facts = {}
```